charts/aad-pod-identity/templates/identities.yaml

```diff
@@ -1,24 +1,26 @@
-{{- if .Values.azureIdentity.enabled }}
-apiVersion: "aadpodidentity.k8s.io/v1"
-kind: AzureIdentity
-metadata:
-  name: {{ .Values.azureIdentity.name }}
-  namespace: {{ template "aad-pod-identity.azureidentity.namespace" . }}
-  labels:
-    {{- include "aad-pod-identity.labels" . | nindent 4 }}
-spec:
-  type: {{ .Values.azureIdentity.type }}
-  ResourceID: {{ required ".Values.azureIdentity.resourceID is required!" .Values.azureIdentity.resourceID }}
-  ClientID: {{ required ".Values.azureIdentity.clientID is required!" .Values.azureIdentity.clientID }}
----
-apiVersion: "aadpodidentity.k8s.io/v1"
-kind: AzureIdentityBinding
-metadata:
-  name: {{ .Values.azureIdentityBinding.name }}
-  namespace: {{ template "aad-pod-identity.azureidentitybinding.namespace" . }}
-  labels:
-    {{- include "aad-pod-identity.labels" . | nindent 4 }}
-spec:
-  AzureIdentity: {{ .Values.azureIdentity.name }}
-  Selector: {{ required ".Values.azureIdentityBinding.selector is required!" .Values.azureIdentityBinding.selector }}
+apiVersion: v1
+kind: List
+items:
+{{- range .Values.azureIdentities }}
+  - apiVersion: "aadpodidentity.k8s.io/v1"
+    kind: AzureIdentity
+    metadata:
+      name: {{ .name }}
+      namespace: {{ default $.Release.Namespace .namespace }}
+      labels:
+        {{- include "aad-pod-identity.labels" $ | nindent 8 }}
+    spec:
+      type: {{ .type }}
+      ResourceID: {{ required "resourceID is required!" .resourceID }}
```

Comment:
The client-go version has been updated to latest in 1.6 and this means the CRD fields are now case sensitive. More details on the changes required - https://github.com/Azure/aad-pod-identity#v160-breaking-change.

Can you update the fields to use the right case when you get a chance?

Reply:
Done